### PR TITLE
DEX-1195 && DEX-1183 - Show urls in jenkins

### PR DESCRIPTION
**Pull request by ngadiyak**

### CODE REVIEW

File: jenkins/tests-pre-release.Jenkinsfile [hunk 1]
```diff
@@ -64,13 +64,17 @@ pipeline {
                 kafkaBuildNumber = Jenkins.instance.getItemByFullName('Waves.Exchange/Matcher/Matcher Server - OS - Test - Kafka').getLastBuild().getNumber() + 1
                 multipleBuildNumber = Jenkins.instance.getItemByFullName('Waves.Exchange/Matcher/Matcher Server - OS - Test - Multiple Versions').getLastBuild().getNumber() + 1
                 versionBuildNumber = Jenkins.instance.getItemByFullName('Waves.Exchange/Matcher/Matcher Server - OS - Test - Version').getLastBuild().getNumber() + 1
+                smokeNBuildNumber = Jenkins.instance.getItemByFullName('Waves.Exchange/Matcher/Matcher Server - OS - Test - Version').getLastBuild().getNumber() + 1
+                smokeNBuildNumber = Jenkins.instance.getItemByFullName('Waves.Exchange/Matcher/Matcher Server - OS - Test - Version').getLastBuild().getNumber() + 1
```
Comment (vsuharnikov) [inline jenkins/tests-pre-release.Jenkinsfile:68]: This lines look similar and it seems `getLastBuild().getNumber()` will be the same for each line. Did you test this?

File: jenkins/tests-pre-release.Jenkinsfile [hunk 2]
```diff
@@ -64,13 +64,17 @@ pipeline {
...
 
                 kafkaBuild = "<a href='/job/Waves.Exchange/job/Matcher/job/Matcher Server - OS - Test - Kafka/${kafkaBuildNumber}'>Kafka</a>"
                 multipleBuild = "<a href='/job/Waves.Exchange/job/Matcher/job/Matcher Server - OS - Test - Multiple Versions/${multipleBuildNumber}'>Multiple</a>"
                 versionBuild = "<a href='/job/Waves.Exchange/job/Matcher/job/Matcher Server - OS - Test - Version/${versionBuildNumber}'>Version</a>"
-                
+                smokeMBuild = "<a href='/job/Waves.Exchange/job/Matcher/job/Matcher Server - OS - Test - Smoke/${smokeNBuildNumber}'>Smoke M</a>"
```
Comment (vsuharnikov) [inline jenkins/tests-pre-release.Jenkinsfile:73]: > Smoke/${smoke**N**BuildNumber}

->

> Smoke/${smokeMBuildNumber}

**Review by vsuharnikov — CHANGES_REQUESTED** [review]
Also it was expected in DEX-1195 that we will display links to harbor in PreRelease job.

File: jenkins/tests-pre-release.Jenkinsfile (hunk 1 above)
Comment (ngadiyak) [reply jenkins/tests-pre-release.Jenkinsfile]: fixed, thx

**Review by vsuharnikov — APPROVED** [review]

**Pull request merged**